packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -184,7 +188,7 @@ export const updatePagesJSToCurrentVersion = async () => {
  */
 export const updateDevDependencies = async () => {
   await updatePackageDependency("@vitejs/plugin-react", null);
-  await updatePackageDependency("vite", "^5.0.10");
+  await updatePackageDependency("vite", null);
```

Comment:
not sure why this changed to null, besides that lgtm

Reply:
It's technically out of scope of my changes but it's a small enough one. It's so the latest version of Vite is pulled instead of a specific one.